backend/tilavarauspalvelu/integrations/email/template_context/common.py

```diff
@@ -261,24 +261,23 @@ def params_for_price_range_info(*, reservation: Reservation) -> dict[str, Any]:
 
 
 def params_for_access_code_reservation(*, reservation: Reservation) -> dict[str, Any]:
+    empty_response = {
+        "access_code_is_used": False,
+        "access_code": "",
+        "access_code_validity_period": "",
+    }
+
     if not reservation.access_code_should_be_active:
-        return {
-            "access_code_is_used": False,
-            "access_code": "",
-            "access_code_validity_period": "",
-        }
+        return empty_response
 
     try:
         response = PindoraClient.get_reservation(reservation=reservation)
     except PindoraNotFoundError:
-        response = None  # Set as None to make mocking in tests easier
-    if not response:
-        # Reservation should have an access code, but it is not available.
-        return {
-            "access_code_is_used": True,
-            "access_code": "",
-            "access_code_validity_period": "",
-        }
+        return empty_response
```

Comment:
I think the previous version was correct, should return this when the code is not found or is inactive in Pindora but Varaamo thinks it should be active (i.e., `reservation.access_code_should_be_active = True`):
`"access_code_is_used": True, "access_code": ""`
Thus the email will instruct the user about the access code requirement but the actual code value is not shown if it is (unexpectedly) not usable.

Reply:
Added the same fix to section access codes